lib/init/vcs.js

```diff
@@ -73,14 +112,16 @@ async function clone(repoUrl, cloneToDir = '.', checkout) {
   repoUrl = repoUrl.replace(/'^\/+|\/+$/g, '');
 
   debug('repoDir is %s', repoDir);
-  await promptForExistingPath(repoDir, `You've downloaded ${repoDir} before. Is it okay to delete and re-download it?`, true);
-  console.log('start cloning...');
-  spawnSync(repoType, cloneArguments(repoType, repoUrl, outputDir), { cmd: repoDir, stdio: 'inherit' });
-  console.log('finish cloning.');
-
-  if (checkout) {
-    debug('checkout is %s', checkout);
-    spawnSync(repoType, ['checkout', checkout], { cmd: repoDir, stdio: 'inherit' });
+  if (isVCSInstalled(repoType)) {
+    cloneRepo(repoType, repoUrl, outputDir, repoDir, checkout);
+  } else {
+    if (repoType === 'git') {
+      console.warn(yellow(`${repoType} is not installed.`));
```

Comment:
`git` command is not installled, try to download template by HTTP.

Reply:
done